packages/react/src/components/table/Table.test.tsx

```diff
@@ -0,0 +1,55 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { axe } from 'jest-axe';
+
+import { DataTable } from './DataTable';
+
+describe('<Table /> spec', () => {
+  let cellConfig;
+  let rows;
+  let caption;
+
+  beforeEach(() => {
+    cellConfig = {
+      cols: [
+        { key: 'id', headerName: 'Not rendered' },
+        { key: 'firstName', headerName: 'First name' },
+        { key: 'surname', headerName: 'Surname' },
+        {
+          key: 'age',
+          headerName: 'Age',
+          transform: ({ age }) => {
+            return <div style={{ textAlign: 'right' }}>{age}</div>;
+          },
+        },
+        { key: 'profession', headerName: 'Profession' },
+      ],
+      indexKey: 'id',
+      renderIndexCol: false,
+    };
+
+    rows = [
+      { id: 1000, firstName: 'Lauri', surname: 'Kekkonen', age: 39, profession: 'Engineer' },
+      { id: 1001, firstName: 'Maria', surname: 'Sarasoja', age: 62, profession: 'Designer' },
+      { id: 1002, firstName: 'Anneli', surname: 'Routa', age: 50, profession: 'Meteorologist' },
+      { id: 1003, firstName: 'Osku', surname: 'Rausku', age: 18, profession: 'Mail Carrier' },
+    ];
+
+    caption = (
+      <span>
+        <b>Table 1</b>: Table description
+      </span>
+    );
+  });
+
+  it('renders the component', () => {
+    const { asFragment } = render(<DataTable caption={caption} cellConfig={cellConfig} rows={rows} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
```

Comment:
I have to fix the generating script at some point. It does not add a line break between test cases.

Reply:
- [ ] Add empty row between unit tests to increase readability